build.gradle.kts

```diff
@@ -61,47 +43,81 @@ for (s in arrayOf(sourceSets.main, sourceSets.test)) {
 }
 
 configurations {
-    val library = this.maybeCreate("library")
+    val library = maybeCreate("library")
     api.configure {
         extendsFrom(library)
     }
 }
 minecraft.runs.all {
     lazyToken("minecraft_classpath") {
-        return@lazyToken configurations.getByName("library").copyRecursive().resolve()
+        return@lazyToken configurations["library"].copyRecursive().resolve()
             .joinToString(File.pathSeparator) { it.absolutePath }
     }
 }
 
 dependencies {
-    minecraft("net.minecraftforge:forge:1.19-41.0.1")
-
-    library("org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version")
-    library("org.jetbrains.kotlin:kotlin-reflect:$kotlin_version")
-    library("org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutines_version")
-    library("org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm:$coroutines_version")
-    library("org.jetbrains.kotlinx:kotlinx-coroutines-jdk8:$coroutines_version")
-    library("org.jetbrains.kotlinx:kotlinx-serialization-json:$serialization_version")
+    minecraft("net.minecraftforge:forge:1.19-41.0.91")
+
+    val library = configurations["library"]
+
+    library("org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version") {
+        exclude(group = "org.jetbrains", module = "annotations")
+        jarJar(group = group!!, name = name, version = "[$version, 2.0)") {
```

Comment:
Why is the version negotiation up to 2.0 only?

Reply:
Now different libs use different versions